tabbycat/utils/management/base.py

```diff
@@ -202,11 +202,12 @@ def get_rounds(self, options):
     def _confirm_rounds(self, rounds, **options):
         if not options["confirm"]:
             self.stdout.write(self.style.WARNING("WARNING! You are about to {} from the following rounds:".format(self.confirm_round_destruction)))
+
             for r in rounds:
                 self.stdout.write(self.style.WARNING("  [{t}]: {r}".format(
                     t=r.tournament.name, r=r.name)))
-            response = input("Are you sure? [y/N]").lower()
-            if response != "yes" and response != "y":
+
+            if input("Are you sure? [y/N]").lower() not in ["yes", "y"]:
```

Comment:
Removing the `response` variable made codeclimate green. Happy to revert if that's not a concern